common/frame.go

```diff
@@ -1859,7 +1859,7 @@ func (f *Frame) evaluate(
 	}
 	eh, err := ec.eval(apiCtx, opts, pageFunc.ToString().String(), evalArgs...)
 	if err != nil {
-		return nil, fmt.Errorf("frame cannot evaluate: %w", err)
+		return nil, fmt.Errorf("%w", err)
```

Comment:
```suggestion
		return nil, err
```

Reply:
Yeah, as mentioned below, we're wrapping it to avoid a `wrapcheck` linter error.